scripts/next-data/generatePreBuildFiles.mjs

```diff
@@ -29,6 +34,25 @@ export const generateBlogYearPages = cachedBlogData =>
     .then(data => [...new Set(data)])
     .then(data => data.forEach(createBlogYearFile));
 
+/**
+ * Render markdown to html via mdx
+ * @param {string} body
+ * @returns {Promise<string>}
+ * @see {https://github.com/mdx-js/mdx/discussions/1985}
+ */
+export const renderMarkdown = async body => {
+  const { default: mdx } = await evaluate(body, {
+    ...runtime,
+    format: "md",
+    development: false,
+    // format: 'md' — treat file as plain vanilla markdown
+    // Need to add the following remark plugins to support GFM and frontmatter
+    // https://github.com/remarkjs/remark-gfm
+    // https://github.com/remarkjs/remark-frontmatter
+    remarkPlugins: [remarkGfm, remarkFrontmatter],
+  })
+  return renderToString(createElement(mdx))
+}
```

Comment:
I referred to https://github.com/mdx-js/mdx/discussions/1985, but there may be a better way.

Reply:
To covert all the ".md" files into HTML pages is a good choice, because an HTML page is easy to read other than the converted js files. +1 for you.

However there're many complicated things to cope with if you do that.

E.g:

'mdx' has already wrapped some functions of render packages. So do  you mean 'mdx' cannot satisfy what you need so you want to remove this reference and change it to 'remark' directly? Nothing worried but just for a confirmation :)